### DISCUSSION TITLE
Welcome to Community Discussions!

### DISCUSSION THREAD

**Discussion by ChristopherA** [ChatTheatre/Community#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gelrir** [comment]
Hi there! I'm Michael Blum, and I used to work at Skotos, making maps and text descriptions, etc. Currently I'm helping Chris with the upgrades to the Gables chat theater.

**Comment by noahgibbs** [comment]
Hello! I'm Noah Gibbs. I'm helping on the technical side. The idea is that over time I'll become the ShannonA replacement in various areas.

**Comment by anony-0** [comment]
Hello, new to this group, new to github, and not even entirely sure I am commenting in the correct place. But anyway, I'm the fellow who made Lazarus Project (Michael), and Chris invited me to make an acct and check out this discussion. I'd like to say I plan to contribute, but I have a feeling I'll mostly be contributing lots of questions.

**Comment by noahgibbs** [comment]
Questions are good!

**Comment by ViciousKoi** [comment]
Hello! Name's Dakota, and I used to play on several of Skotos' games. I have some backend web development under my belt, but have never approached the beast of coding for chat theatre before.

I'll be here trying to start my own project based off a world I've been building over the last 4 years, and possibly getting involved in others as my own skill develops.

**Reply by shentino** [reply]
> Hello! Name's Dakota, and I used to play on several of Skotos' games. I have some backend web development under my belt, but have never approached the beast of coding for chat theatre before.
> 
> I'll be here trying to start my own project based off a world I've been building over the last 4 years, and possibly getting involved in others as my own skill develops.

Is that you Virendra?

**Reply by ViciousKoi** [reply]
> Is that you Virendra?

been a few years since I went by that! But yeah. Dakota's fine now. :)

**Reply by shentino** [reply]
I was Muriku

Man it's been ages.  I'd like to hook up sometime for a chat, you cool with that?

**Reply by ViciousKoi** [reply]
Yeah dude no problem. GitHub doesn't seem to send me notifications when I get a reply here. If you have discord, my name there is Sparky Tundabolt#0518

**Reply by shentino** [reply]
> > Is that you Virendra?
> 
> been a few years since I went by that! But yeah. Dakota's fine now. :)

The name "ViciousKoi" kinda gave it away ;)

**Reply by shentino** [reply]
> Yeah dude no problem. GitHub doesn't seem to send me notifications when I get a reply here. If you have discord, my name there is Sparky Tundabolt#518

Added.  You left out a leading zero in the number part, just waiting on your end now

**Comment by noahgibbs** [comment]
Awesome! Glad to have you here.

**Reply by shentino** [reply]
> Awesome! Glad to have you here.

I second the motion, I knew Virendra as an awesome RPer back in ironclaw, definitely has the passion

**Comment by shentino** [comment]
I'm shentino, I got the alpha version of SkotOS released, and I also make my own mudlib.
I also sadly miss Ironclaw Online.

**Comment by ChristopherA** [comment]
@ViciousKoi — welcome to the community!

You said you were a backend web developer — are you interested in possibly helping out with some of those aspects of Chat Theatre's that use SkotOS?

Do you use a Macintosh? By chance have expertise in PHP?

**Reply by ViciousKoi** [reply]
Time got away from me. Sorry for the late reply lol. Went to school for web development, but never did any paid work for backend stuff. Found out that I liked programming more as a hobby than a career. 

I do not have a Mac, but I have a bit of experience with PHP. I've built a simple database before.

**Comment by shentino** [comment]
@ViciousKoi are you still around?
